SwrveCommonSDK/src/main/java/com/swrve/sdk/messaging/view/SwrveDialog.java

```diff
@@ -71,4 +74,11 @@ private void goneAway() {
             }
         }
     }
+
+    public Activity getParentActivity() {
+        if (parentActivity != null) {
+            return parentActivity.get();
+        }
+        return null;
+    }
 }
```

Comment:
Does the following do this anyway? http://developer.android.com/reference/android/app/Dialog.html#getOwnerActivity()